src/functional/ffront/type_info.py

```diff
@@ -286,6 +312,37 @@ def is_concretizable(symbol_type: ct.SymbolType, to_type: ct.SymbolType) -> bool
     return False
 
 
+def _is_empty_field(field: ct.SymbolType) -> bool:
+    return isinstance(field, ct.FieldType) and len(field.dims) == 0
+
+
+def is_not_empty_field_compatible(a_arg: ct.SymbolType, b_arg: ct.SymbolType) -> bool:
+    """
+    Check if first argument is an empty field and whether second is either another emtpy field or a scalar.
+
+    In both cases, arguments dtypes have to be the same
+    Examples:
+    ---------
+    >>> is_not_empty_field_compatible(
```

Comment:
When all examples return `True` they give little insight into what the function does.

Reply:
ok, I can add a couple when it outputs False